docs/appendices/EOL.rst

```diff
@@ -1,6 +1,11 @@
 End of life statements
 ======================
 
+We aim to have a release every six months.
+The latest and previous release receive correctness, stability and security updates.
+The release before that gets critical security updates only.
```

Comment:
```suggestion
The second-to-last release before that gets critical security updates only.
```

Reply:
I find second-to-last always confusing. To me it means the one before last here.